### PR TITLE
feat: /stack and /protocol pages

### PR DESCRIPTION
**Pull request by mohandast52**
- Updated "Protocol" & "Stack" page

https://github.com/valory-xyz/olas-website/assets/22061815/bd5d2f3a-6d5e-42fe-b261-83c9517e9839

https://github.com/valory-xyz/olas-website/assets/22061815/679d673d-c4a7-4ea9-8262-0ee6e1191c31

### CODE REVIEW

File: pages/stack.jsx [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
+import { StackCta } from 'components/StackPage/StackCta';
+import { StackKeyFeatures } from 'components/StackPage/StackKeyFeatures';
+import { StackFaq } from 'components/StackPage/StackFaq';
 
 const StackPage = () => (
   <PageWrapper>
-    <SectionWrapper customClasses="px-4 py-12 border-b">
-      <div className={SCREEN_WIDTH_LG}>
-        <h2 className={`${SUB_HEADER_CLASS} text-left mb-8`}>Olas Stack</h2>
+    <Meta pageTitle="Stack" description="Explore Olas Stack Today" />
```
[the remark this answers is not in the document]
Comment (oaksprout) [reply pages/stack.jsx:10]: That's fine, thanks

File: pages/protocol.jsx [hunk 2]
```diff
@@ -1,27 +1,18 @@
...
+import { ProtocolKeyFeatures } from 'components/ProtocolPage/ProtocolKeyFeatures';
+import { ProtocolCta } from 'components/ProtocolPage/ProtocolCta';
+import { WhyOlasProtocol } from 'components/ProtocolPage/WhyOlasProtocol';
 
 const ProtocolPage = () => (
   <PageWrapper>
-    <SectionWrapper customClasses="px-4 py-12 border-b">
-      <div className={SCREEN_WIDTH_LG}>
-        <h2 className={`${SUB_HEADER_CLASS} text-left mb-8`}>Olas Protocol</h2>
+    <Meta pageTitle="Olas Protocol" description="Explore Olas Protocol Today" />
```
[the remark this answers is not in the document]
Comment (oaksprout) [reply pages/protocol.jsx:10]: also fine

**Review by Tanya-atatakai — DISMISSED** [review]

**Review by oaksprout — APPROVED** [review]

**Review by truemiller — APPROVED** [review]

File: pages/stack.jsx [hunk 3]
```diff
@@ -1,30 +1,18 @@
...
-            className="text-purple-600"
-          >
-            Go to Documentation
-          </Link>
-        </p>
-      </div>
-    </SectionWrapper>
+    <StackHeader />
+    <StackKeyFeatures />
+    {/* <StackFaq /> */}
```
Comment (Tanya-atatakai) [inline pages/stack.jsx:14]: should this be commented? I see it in the video you provided

File: pages/protocol.jsx [hunk 4]
```diff
@@ -1,27 +1,18 @@
...
-        <p className={`${TEXT_MEDIUM_CLASS} mb-4`}>
-          <Link href={`${DOCS_BASE_URL}/protocol`} className="text-purple-600">
-            Go to Documentation
-          </Link>
-        </p>
-      </div>
-    </SectionWrapper>
+    <ProtocolHeader />
+    <ProtocolKeyFeatures />
+    {/* <WhyOlasProtocol /> */}
```
Comment (Tanya-atatakai) [inline pages/protocol.jsx:14]: should this also be commented?

**Review by Tanya-atatakai — APPROVED** [review]

**Pull request merged**